desktop/core/src/desktop/conf.py

```diff
@@ -2058,6 +2061,14 @@ def default_gist_preview():
   type=coerce_bool,
   help=_('Turn on the direct link sharing of saved document.')
 )
+
+USE_THRIFT_HTTP_JWT = Config(
+  key="use_thrift_http_jwt",
+  help=_("Use Thrift over HTTP for the transport mode with JWT as Bearer header."),
```

Comment:
"Use JWT as Bearer header for authentication when using Thrift over HTTP transport"

More explicit?